src/jdk.management/share/classes/com/sun/management/OperatingSystemMXBean.java

```diff
@@ -130,18 +130,30 @@ public interface OperatingSystemMXBean extends
     public long getTotalMemorySize();
 
     /**
-     * Returns the "recent cpu usage" for the whole system. This value is a
+     * Returns the "recent CPU usage" for the whole system. This value is a
      * double in the [0.0,1.0] interval. A value of 0.0 means that all CPUs
      * were idle during the recent period of time observed, while a value
      * of 1.0 means that all CPUs were actively running 100% of the time
-     * during the recent period being observed. All values between 0.0 and
-     * 1.0 are possible depending of the activities going on in the system.
-     * If the system recent cpu usage is not available, the method returns a
+     * during the recent period being observed.
+     *
+     * The recent period of observation is implementation-specific, and
+     * typically relates to the duration since the last call made to this
+     * method, or {@link #getCpuLoad()}. For the very first invocation, the
+     * recent period of observation is undefined.
+     *
+     * All values between 0.0 and 1.0 are possible dependent on the activities
+     * going on. If the recent CPU usage is not available, the method returns a
      * negative value.
      *
      * @deprecated Use {@link #getCpuLoad()} instead of
      * this historically named method.
      *
+     * @apiNote Callers should be aware of the possibility of other callers
+     * affecting the observation period and the result.
+     *
+     * @implNote There is only one observation period for the entire JVM
```

Comment:
This version looks good, I'm just wondering about the implNote, is the statement true for all implementations (Linux, macOS, Windows, ...)?

Reply:
Reviewing the current implementation this seems to be the case. 

The `@implNote` was more meant as an explanation to why the result could be affected by other callers. Since the `@apiNote` also states that the "observation period and result" can be affected, I think its an unnecessary implementation note to have.

Reply:
Okay, if this is the case for all implementations then it would be okay to use "In the JDK implementation, there is only one ...".

Reply:
Then again, I don't know the purpose the comment serves. 

For Windows, the observation period is at least 500 ticks. There is however still only one observation period, meaning any caller using this method will change that observation period.